deploy/helm/grafana-operator/values.yaml

```diff
@@ -77,12 +86,17 @@ podAnnotations: {}
 # -- pod security context
 podSecurityContext: {}
 
-# -- grafana operator container security context
+# https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
```

Comment:
```suggestion
```
Let's remove it altogether. - It's definitely useful to remove `--` as, otherwise, the generated README.md is cluttered with a complex object. The link is not needed either for various reasons: external links tend to change and `securityContext` is a well-known concept anyway.